redaxo/src/addons/media_manager/lib/managed_media.php

```diff
@@ -322,6 +330,8 @@ protected function getImageSource()
         } elseif ('webp' == $format) {
             $quality = (int) $this->getImageProperty(self::PROP_WEBP_QUALITY, $addon->getConfig('webp_quality'));
             imagewebp($this->image['src'], null, $quality);
+        } elseif ('avif' == $format) {
+            imageavif($this->image['src']);
```

Comment:
Müssen wir für avif die quality auch einstellbar machen? Also will man da teils auch vom Default abweichen?

Reply:
Ich mache mich gerade kundig.

avif kann ja statische und animierte Bilder, mit und ohne Transparenz mit Qualitätsabstufungen. Ich nehme an, es soll erstmal nur um statische Bilder gehen.

https://reachlightspeed.com/blog/using-the-new-high-performance-avif-image-format-on-the-web-today/

https://php.watch/versions/8.1/gd-avif

Es müsste von den basics her quality und speed einstellbar sein:

https://www.php.net/manual/en/function.imageavif.php